### PR TITLE
bootstrap: add solus

### PR DESCRIPTION
**Pull request by kyrios123**
Add support for Solus distribution.

Signed-off-by: Pierre-Yves <pyu@riseup.net>

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -30,6 +30,10 @@ for i in $ID $ID_LIKE; do
 			PKG_CONFIG_32=i586-slackware-linux-gnu-pkg-config
 			PKG_CONFIG_64=x86_64-slackware-linux-gnu-pkg-config
 			;;
+		solus)
+			PKG_CONFIG_32=pkg-config
```
Comment (dhewg) [inline bootstrap.sh]: If there's `x86_64-solus-linux-gnu-pkg-config`, isn't there a i*86 specific version too instead of the generic `pkg-config`?

Comment (kyrios123) [reply bootstrap.sh]: As far as I know only the generic pkg-config is available for 32bit.
Currently I use the [same patch](https://dev.getsol.us/differential/changeset/?ref=45213&whitespace=ignore-most) for building wine-nine-standalone on Solus.

Comment (dhewg) [reply bootstrap.sh]: That's unusual, but okay then

**Comment by dhewg** [comment]
Thanks!

**Pull request merged**